.github/workflows/coverage.yml

```diff
@@ -36,9 +36,8 @@ env:
 
 jobs:
   coverage:
-    name: Coverage nightly
+    name: Coverage on stable
     # The large timeout is to accommodate:
-    # - nightly builds (75 minutes, typically 30-50 minutes), and
```

Comment:
Instead of deleting we should have something as:

# - stable builds (typically 30-50 minutes), and

Reply:
Included in [fix(ci): timeout and docs](https://github.com/ZcashFoundation/zebra/pull/4465/commits/9f6ddb7bb9082a8084b1933a9075f15a7cb41d04)